styles/styles.go

```diff
@@ -58,13 +59,19 @@ func SelectTheme(theme string) glamour.TermRendererOption {
 	case "notty":
 		return glamour.WithStyles(glamour.NoTTYStyleConfig)
 	default:
-		if termenv.EnvNoColor() {
-			return glamour.WithStyles(glamour.NoTTYStyleConfig)
-		}
+		bytes, err := os.ReadFile(theme)
+		if err != nil {
+			// Should log a warning so the user knows we failed to read their theme file
+			if termenv.EnvNoColor() {
+				return glamour.WithStyles(glamour.NoTTYStyleConfig)
+			}
 
-		if !termenv.HasDarkBackground(){
-			return glamour.WithStyles(glamour.LightStyleConfig)
+			if !termenv.HasDarkBackground() {
+				return glamour.WithStyles(glamour.LightStyleConfig)
+			}
+			return glamour.WithStylesFromJSONBytes(DefaultTheme)
 		}
-		return glamour.WithStylesFromJSONBytes(DefaultTheme)
+
+		return glamour.WithStylesFromJSONBytes(bytes)
```

Comment:
We should just return this if `err != nil` to remove a layer of nesting

Reply:
Done